src/Controllers/Db_auth.php

```diff
@@ -47,7 +47,7 @@ public function userAuth($twig)
             if (password_verify($password, $hashed_password)) {
                 $auth_user = $user->getUsername();
 
-                session_start();
+                // session_start();
```

Comment:
Why did you comment this out??

Reply:
Was just checking if it resolved the session already start error log, it did but then this issue #204 was occurring

Reply:
Okay. Got it!